modules/sff/module/inc/sff/sff.x

```diff
@@ -59,6 +59,9 @@ SFF_MEDIA_TYPE_ENTRY(1G_BASE_CX, 1GBASE-CX)
 SFF_MEDIA_TYPE_ENTRY(1G_BASE_T, 1GBASE-T)
 SFF_MEDIA_TYPE_ENTRY(100_BASE_LX, 100BASE-LX)
 SFF_MEDIA_TYPE_ENTRY(100_BASE_FX, 100BASE-FX)
+SFF_MEDIA_TYPE_ENTRY(100G_BASE_LR, 100GBASE-LR)
```

Comment:
You should group the 100G enums together.

Reply:
grouped 100G...